frontend/front/src/dummyData/jsonServerData/property_assessments.js

```diff
@@ -0,0 +1,70 @@
+module.exports = [
+  {
+    created_at: null, // type: datetime, null: false
```

Comment:
Good call with these type comments IMO. Make sure all these new guys have an `id` just for json-server's sake